apps/emqx_utils/src/emqx_placeholder.erl

```diff
@@ -254,14 +271,26 @@ bin(Val) -> emqx_utils_conv:bin(Val).
 quote_sql(Str) ->
     emqx_utils_sql:to_sql_string(Str, #{escaping => sql, undefined => <<"undefined">>}).
 
+-spec quote_sql2(_Value) -> iolist().
+quote_sql2(Str) ->
+    emqx_utils_sql:to_sql_string(Str, #{escaping => sql}).
+
```

Comment:
Nit. Naming is far from obvious, maybe `quote_sql_bwcompat/1` for the former and `quote_sql/1` for the latter?